docs/user/reference/features.rst

```diff
@@ -1,7 +1,56 @@
-========
-Features
-========
+=================
+Feature reference
+=================
 
+.. TODO: Continue to add more features here.
+
+Hosting features
+----------------
+
+⏩️ :doc:`/custom-domains`
+  Documentation projects can use their own domain name.
+  A project may define multiple domains,
+  as well as define the *canonical domain* where all other domains will redirect.
+
+
+⏩️ :doc:`/versions`
+  We support multiple versions and translations,
+  integrated nicely into the URL of your documentation.
+  This is served at ``/en/latest/`` by default.
+  If you only have 1 version and translation,
+  we also support :doc:`single version projects </single_version>` served at ``/``.
```

Comment:
This is a really nice purposeful use of the introduction, clarifying a basic misconception that somehow multiple versions are required :+1: